mlir/test/CAPI/mixr_full.c

```diff
@@ -165,6 +166,20 @@ static bool constructAndTraverseIr(MlirContext ctx) {
   mlirMIGraphXAddHighLevelPipeline(pm);
   mlirPassManagerRun(pm, module);
   MlirOperation moduleOp = mlirModuleGetOperation(module);
+
+  MlirRockTuningSpace tuningSpace = mlirRockTuningSpaceCreate(module);
+  int qNum = mlirRockTuningGetNumParamsQuick(tuningSpace);
+  int fNum = mlirRockTuningGetNumParamsFull(tuningSpace);
+  MlirRockTuningParam obtainedParam =
+      mlirRockTuningCreateParamAt(tuningSpace, 0);
+  mlirRockTuningSetParam(module, obtainedParam);
+  MlirStringRef paramStr = mlirRockTuningGetParamStr(obtainedParam);
+  mlirRockTuningParamDestroy(obtainedParam);
+  mlirRockTuningSpaceDestroy(tuningSpace);
+
+  printf("Got tuning space,\n");
+  printf("Obtained perfconfig : \"%s\"\n", paramStr.data);
+  printf("quick set = %d, full set = %d\n", qNum, fNum);
```

Comment:
Yeah, that should be done in the next step, I'm planning to expose some functionalities to a standalone tool or as rocmlir-gen features so that we can more easily include it in our infrastructure